enhancements/authentication/configuring-webhook-token-authenticators.md

```diff
@@ -85,6 +85,9 @@ configure certificate authority, client certificate and key.
 This path is compatible with kubernetes as it is possible to directly map the referenced
 secret's contents to files and specify these in the kube-apiserver configuration.
 
+The new `spec.webhookTokenAuthenticator` structure gets extended by the `cacheTTL time.Duration`
+field that maps to the `--authentication-token-webhook-cache-ttl` option of the kube-apiserver.
```

Comment:
Is this coming from an RFE?  This seems like a knob for the sake of a knob.

Reply:
It does not, but since this is a problem for us, I would expect it to be a problem for other IdPs implementing token timeouts their own way.